test/suites/container_devices_infiniband_physical.sh

```diff
@@ -0,0 +1,105 @@
+# Activating Infiniband VFs:
+# Mellanox example:
+# wget http://www.mellanox.com/downloads/ofed/MLNX_OFED-4.6-1.0.1.1/MLNX_OFED_LINUX-4.6-1.0.1.1-ubuntu18.04-x86_64.tgz
+# tar zxvf MLNX_OFED_LINUX-4.6-1.0.1.1-ubuntu18.04-x86_64.tgz
+# cd MLNX_OFED_LINUX-4.6-1.0.1.1-ubuntu18.04-x86_64/
+# sudo ./mlnxofedinstall  --force
+# sudo mlxconfig --yes -d /dev/mst/mt4099_pciconf0 set LINK_TYPE_P2=2
+# echo "options mlx4_core num_vfs=4 probe_vf=4" | sudo tee /etc/modprobe.d/mellanox.conf
+# reboot
+test_container_devices_ifb_physical() {
+  ensure_import_testimage
+  ensure_has_localhost_remote "${LXD_ADDR}"
+
+  parent=${LXD_IFB_PHYSICAL_PARENT:-""}
```

Comment:
- standard abbreviation of infiniband is `IB`
 - New variable should be documented in test/README.md